src/static/js/modules/loan-comparison/api.js

```diff
@@ -36,5 +41,20 @@ function prepLoanData(loan) {
     }
 }
 
-
+function prepLoanDataForMtgIns(loan) {
+    var minfico = parseInt(loan['credit-score']) || 0;
```

Comment:
This plus `maxfico` would be useful as functions since they're used 2x, although we'll probably want to refactor this function with prepLoanData() also..not urgent tho.